servo-core/src/main/java/com/netflix/servo/util/Preconditions.java

```diff
@@ -39,13 +39,19 @@ private Preconditions() {
    * Ensures the truth of an expression involving one or more parameters to the
    * calling method.
    *
+   * @param errorMessage the error message - can be a <a href="../util/Formatter.html#syntax">format string</a>
+   * @param args arguments if using a formatted string
    * @param expression a boolean expression
    * @throws IllegalArgumentException if {@code expression} is false
    */
-  public static void checkArgument(
-      boolean expression, String errorMessage) {
+  public static void checkArgument(boolean expression, String errorMessage, String... args) {
```

Comment:
@dmuino any compatibility concerns here? This is changing the signature on a public api.